docs/Migration.md

```diff
@@ -0,0 +1,35 @@
+# Migrate an existing Android app to AEP Android SDK 2.0
+
+This document describes how existing Android SDK 1.x customers can upgrade to the Android SDK 2.0 without introducing any breaking changes. We have made upgrading seamless, which will ensure that APIs between SDK 1.x and 2.0 are compatible.
```

Comment:
nit: I wonder if calling the sdk Android SDK 2.0 will cause confusion for implementers. I cannot think of another name that fits here though! 😅 . Do you think adding a maven hyperlink to these SDK's will clarify it? Your call.

Reply:
We will show maven links for core extensions on the main README. So, I think it should be good.